types/ibmcloud-appid/index.d.ts

```diff
@@ -0,0 +1,69 @@
+// Type definitions for ibmcloud-appid 6.2
+// Project: https://github.com/ibm-cloud-security/appid-serversdk-nodejs
+// Definitions by: Younes A <https://github.com/younes-io>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface StrategyOptions {
+    [key: string]: any;
+}
+
+export interface SelfServiceOptions {
+    iamApiKey?: string;
+    managementUrl?: string;
+    tenantId?: string;
+    oAuthServerUrl?: string;
+    iamTokenUrl?: string;
+}
+
+export class Strategy {
+    authenticate: () => void;
+}
+
+export interface ApplicationIdentityToken {
+    accessToken: string;
+    tokenType: string;
+    expiresIn: number;
+}
+
+export interface CustomIdentityToken extends ApplicationIdentityToken {
+    identityToken: string;
+}
+
+export interface UserSCIM {
+    id: string;
+    userName: string;
+    [key: string]: any;
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class APIStrategy extends Strategy {
+    constructor(options: StrategyOptions);
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class WebAppStrategy extends Strategy {
+    constructor(options: StrategyOptions);
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class TokenManager {
+    constructor(options: StrategyOptions);
+    getApplicationIdentityToken: () => Promise<ApplicationIdentityToken | Error>;
+    getCustomIdentityTokens: () => Promise<CustomIdentityToken | Error>;
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class SelfServiceManager {
+    constructor(options: SelfServiceOptions);
+    signUp: (userData: unknown, language: string, iamToken: string) => Promise<UserSCIM>;
```

Comment:
```suggestion
    signUp: (userData: { [key: string]: unknown }, language: string, iamToken: string) => Promise<UserSCIM>;
```
> Sign up a new user. userData is a JSON object

so use something silar to this keyed object definition, not pain `unknown`